pep-0001.txt

```diff
@@ -275,12 +275,17 @@ The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
 the BDFL-Delegate for that PEP, and they will then have the authority to approve
```

Comment:
Just FYI, if this is re-opened, this line should probably be modified as well, since it mentions only `BDFL-Delegate` in the context of current processes.